vendor/github.com/containers/storage/drivers/chroot_unix.go

```diff
@@ -11,8 +11,8 @@ import (
 // chrootOrChdir() is either a chdir() to the specified path, or a chroot() to the
 // specified path followed by chdir() to the new root directory
 func chrootOrChdir(path string) error {
-	if err := syscall.Chroot(path); err != nil {
-		return fmt.Errorf("error chrooting to %q: %v", path, err)
+	if err := syscall.Chroot(path); err == nil {
+		return nil
```

Comment:
This looks very incorrect; `chdir("/")` is an essential part of the `chroot`ing process, and by no means a replacement for it!

Reply:
yes sorry, I messed up the conditions.  I think the Chroot errors should be ignored, in that case only the Chdir is performed (as it looks it should be from the code comments)